mongodb-atlas-mixin/config.libsonnet

```diff
@@ -0,0 +1,20 @@
+{
+  _config+:: {
+    // sharding dashboard flag
+    enableShardingOverview: false,
+
+    dashboardTags: ['mongodb-atlas-mixin'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+
+    // alerts thresholds
+    alertsDeadlocks: 10,  // count
+    alertsSlowNetworkRequests: 10,  // count
+    alertsHighDiskUsage: 90,  // percentage: 0-100
+    alertsSlowHardwareIO: 3,  // seconds
+    alertsHighTimeoutElections: 10,  // count
+
+    enableLokiLogs: false,
```

Comment:
```suggestion
```
Worth dropping if we don't any logs?

Reply:
can do, wasn't sure if that needed to be there regardless